src/widgets/DataTable/DataTable.js

```diff
@@ -19,7 +20,14 @@ import { StyleSheet, VirtualizedList, VirtualizedListPropTypes } from 'react-nat
 const DataTable = React.memo(({ renderRow, renderHeader, style, ...otherProps }) => (
   <>
     {renderHeader()}
-    <VirtualizedList style={style} renderItem={renderRow} {...otherProps} />
+    <KeyboardAwareScrollView style={style} enabled behaviour="padding" keyboardVerticalOffset={500}>
+      <VirtualizedList
+        keyboardShouldPersistTaps="always"
+        style={style}
+        renderItem={renderRow}
+        {...otherProps}
+      />
+    </KeyboardAwareScrollView>
```

Comment:
What was this fixing?

Reply:
without it, when I went to edit cells that would be underneath the keyboard, the cell would lose focus and the keyboard would dismiss automatically and not let me edit